joomla/joomla-cms
# [4.3] Rename SQL update scripts for 4.3.2

Pull Request for Issue # .

### Summary of Changes

After 4.3.1 has been released, new update SQL scripts "4.3.0-2023-03-31.sql" have been added with pull request (PR) #40258  and new update SQL scripts "4.3.0-2023-05-03.sql” with my PR #40535 .

These script should be named "4.3.2-..." and not "4.3.0-..." as they will be released with 4.3.2.

For the first PR #40258 the reason was because that PR was prepared before 4.2.0 was released, and it has been forgotten to rename the files before merging the PR.

For the 2n PR #40535 I can say that it was my fault, I haven't noticed the wrong version in the file name.

For lucky circumstance the files did not sort somewhere in the middle but still at the end of the files list, so in principle we could leave things as they are, but the wrong names might confuse people.

Therefore we should rename them before 4.3.2 is released.

This is what this PR here does.

Because the scripts contain only update statements which do not do any harm when being executed again, renaming the files will not make problems for people who update between nightlies, which otherwise could be the case. So renaming them is even save for that unsupported use case.

### Testing Instructions

Code review.

Or update a 4.3.1 to the latest 4.3-dev nightly to get the actual result, and update a 4.3.1 to the update package or custom update URL created by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

Scripts have names "4.3.0-2023-03-31.sql" and "4.3.0-2023-05-03.sql" but they will be released with 4.3.2, see milestone and merge date of the 2 mentioned PRs #40258 and #40535 .

When updating a 4.3.1 to the latest 4.3-dev nightly, both scripts are executed.

Database schema version of the core after the update is "4.3.0-2023-05-03".

### Expected result AFTER applying this Pull Request

Scripts have the right names "4.3.2-2023-03-31.sql" and "4.3.2-2023-05-03.sql".

When updating a 4.3.1 to the the update package or custom update URL created by Drone for this PR, both scripts are executed.

Database schema version of the core after the update is "4.3.2-2023-05-03".

### Link to documentations
Please select:
